src/mono/wasm/build/WasmApp.targets

```diff
@@ -63,6 +63,7 @@
       - $(RunAOTCompilationAfterBuild)      - Run AOT compilation even after Build. By default, it is run only for publish.
                                               Defaults to false.
       - $(WasmAotProfilePath)               - Path to an AOT profile file.
+      - $(FeatureWasmLegacyJsInterop)       - Include support for legacy JS interop. Defaults to true.
```

Comment:
Should this be false instead?

Reply:
Yes, after we are sure there are no remaining legacy interop in Blazor itself. 
Or after they flip it to true for Blazor.
Also this is just half of the solution. There is another draft PR for native and JS part. They need to be disabled together.